static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -298,6 +307,22 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
     );
   }
 
+  // TODO: remove once we have mobile replay onboarding
+  if (['android', 'react-native'].includes(currentPlatform.language)) {
+    return (
+      <MobileBetaBanner
+        link={`https://docs.sentry.io/platforms/${currentPlatform.language}/session-replay/`}
+      />
+    );
+  }
+  if (currentPlatform.language === 'apple') {
+    return (
+      <MobileBetaBanner
+        link={`https://docs.sentry.io/platforms/apple/guides/ios/session-replay/`}
+      />
+    );
+  }
```

Comment:
did it this way instead of updating the platform guide file itself bc it was getting too complicated with the npm/loader states and platformOptions that didn't apply in our situation (all we want to do is render a simple blue banner). figured this was a lot simpler, and easier to delete since we'll remove this eventually anyway.